### ISSUE TITLE
Video clips

### ISSUE THREAD

**Issue by FevLoad** [valeriobasile/twittercrawler#7]
Hello there mate
Can this bot download videos and photos from each tweet if I made it monitor or search a certain user?!

Sorry for opening this issue.

**Comment by valeriobasile** [comment]
I don't think so, it was made to download text only, I never checked on the media. You should be able to retrieve the URLs for media content though, unless the API has changed, making this script obsolete.

**Comment by FevLoad** [comment]
Can't be edit to make the script take picture of the tweet instead of download the text
